### PR TITLE
remove conda ruamel package to hopofully fix pipelines ci build

### PR DESCRIPTION
**Pull request by jenshnielsen**
this no longer works well together with the pip version of the package

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -18,7 +18,6 @@ dependencies:
  - pytest-runner
  - spyder
  - pyzmq
- - ruamel_yaml
  - wrapt
  - pyyaml
```
Comment (astafan8) [inline environment.yml:22]: what about this pyyaml ? :)

File: environment.yml [hunk 2]
```diff
@@ -18,7 +18,6 @@ dependencies:
  - pytest-runner
  - spyder
  - pyzmq
- - ruamel_yaml
```
Comment (astafan8) [inline environment.yml:21]: we don't need to include it in the "pip" section becuase it's a requirement in qcodes's setup file, right?

Comment (jenshnielsen) [reply environment.yml:21]: No but that is true for a lot of package in both the env file and the requirements

File: environment.yml (hunk 1 above)
Comment (jenshnielsen) [reply environment.yml:22]: Not really related to this pr but it seems to be unused so lets remove it

**Pull request merged**